plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.tools.eclipse</groupId>
+    <artifactId>trunk</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <relativePath>../../</relativePath>
+  </parent>
+  <groupId>com.google.cloud.tools.eclipse</groupId>
+  <artifactId>com.google.cloud.tools.eclipse.appengine.newproject.test</artifactId>
+  <packaging>eclipse-test-plugin</packaging>
+
+  <build>
+    <sourceDirectory>src</sourceDirectory>
+    <plugins>
+      <plugin>
```

Comment:
Are we likely to need to do that though? I'd like to have the places we need to add things to in place, rather than trying to figure them out down the line when we do need them. Depending on hidden defaults is often confusing.